src/resources/views/crud/inc/form_save_buttons.blade.php

```diff
@@ -6,13 +6,18 @@
             <div class="btn-group" role="group">
```

Comment:
The dropdown button in SaveActions doesn't seem to work. When I click the caret, it doesn't drop down:

![CleanShot 2023-02-26 at 10 53 24](https://user-images.githubusercontent.com/1032474/221401119-5474b2b1-372c-4f0a-91b3-a4470f16c5c4.gif)

Probably a change in HTML syntax for dropdowns?

Reply:
@tabacitu 
I noticed that `bootstrap@5.2.3` uses `popper@2.11.6` but we are using `popper@1.16.1` (base on `src/resources/views/base/inc/scripts.blade.php:2`).

I tried by loading instead `@basset('https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js')` and now the dropdown works... Can it be that we were using the wrong popper version? Suggestions?

**BTW**, the dropdown component did change in bootstrap 5, so that's a additional changed required.